test-resources/go/feature_flag/builtin_rules/statement_cleanup/input/sample.go

```diff
@@ -106,3 +106,57 @@ func after_return4() string {
     fmt.Println("5")
     return "enabled"
 }
+
+
+func simplify_if_statement_false_comment_demo_single_comment() {
+    if exp.BoolValue("false") {
+        fmt.Println("to be removed")
+    } else {
+        fmt.Println("remain")
+    }
+    // this comment doesnt get removed but it should
```

Comment:
No longer true, no? Shouldn't the comment be updated to saying it will get removed (same with all the cases below)